src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -182,7 +182,7 @@ class TezosPlatformDiscoveryOperations(
   /** Method querying slick metadata tables for entities */
   private def preCacheEntities: DBIO[EntitiesCache] = {
     val result = for {
-      tables <- MTable.getTables(Some(""), Some("public"), Some(""), Some(Seq("TABLE")))
+      tables <- MTable.getTables(Some(""), Some("tezos"), Some(""), Some(Seq("TABLE")))
```

Comment:
I am getting the feeling that, these should be parameterized via configuration.

Reply:
Indeed, but maybe we can put those changes with a PR that actually uses the parameterized database namespace everywhere.
Doing it now, would be an isolated change between many that will need be done, anyway.
WDYT?

Reply:
Fair enough